docs/ingestion/dynamodb.md

```
@@ -26,7 +26,7 @@ To connect to DynamoDB, you need to add a configuration item to the connections
 * `secret_access_key`: Password for the IAM account.
 * `region`: AWS region in which your DynamoDB table exists.
 
-For details on how to obtain these credentials, please refer [here](https://bruin-data.github.io/ingestr/supported-sources/dynamodb.html#setting-up-a-dynamodb-integration).
+For details on how to obtain these credentials, please refer [here](https://getbruin.com/docs/ingestr/supported-sources/dynamodb.html#setting-up-a-dynamodb-integration).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)